backend/canisters/storage_bucket/CHANGELOG.md

```diff
@@ -7,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Add cacheable resource header ([#5947](https://github.com/open-chat-labs/open-chat/pull/5947))
```

Comment:
There is already a Changed section so no need to add a new one

Reply:
There are already _lots_ of changed sections. I don't really understand what this log is supposed to be telling us.

Reply:
Oh I think I get it. Release tags interspersed with changed, added etc